<launch>
    <node pkg="ntrip_ros" type="ntripclient.py" name="ntrip_client" output="screen">
        <param name="ntrip_server" value="euref-ip.net:2101" />
        <param name="ntrip_user" value="" />
        <param name="ntrip_pass" value="" />
        <param name="ntrip_stream" value="DARE00GBR0" />
        <param name="nmea_gga" value="GPGGA,%02d%02d%04.2f,5212.9977,N,00653.9845,E,1,16,1.1,48.0,M,94.0,M,0,0" />
        <param name="rtcm_topic" value="/rtcm" />
    </node>
</launch>


